ProcessMaker/screen-builder
# Loading button update

Feature to allow the submit button to turn into a loading spinner instead of a full screen interstitial

## How to Test
- Create a screen and add a submit button. Enable "Loading submit button"
- Create a process with a task for the screen and enable "Display next assigned task to assignee"
- Verify the button turns into a spinner and stays a spinner until the next screen is displayed


## Related Tickets & Packages
- Requires: https://github.com/ProcessMaker/package-webentry/pull/219
- Jira: https://processmaker.atlassian.net/browse/FOUR-10471

## Code Review Checklist

ci:next
ci:package-savedsearch:fix-test-teardown
